### PR TITLE
ncempy: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tschoonj** [comment]
@conda-forge/staged-recipes could anyone review this recipe? Thanks!

File: recipes/ncempy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - h5py
+    - setuptools
+
+test:
+  imports:
+    - ncempy
+
+about:
+  home: https://github.com/ercius/openNCEM
+  license: GPL-3.0
```
Comment (chrisburr) [inline recipes/ncempy/meta.yaml]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/ncempy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - python >=3
+    - numpy
+    - scipy
+    - matplotlib
+    - h5py
+    - setuptools
+
+test:
+  imports:
+    - ncempy
```
Comment (chrisburr) [inline recipes/ncempy/meta.yaml]: Can you also test the command?
```suggestion
    - ncempy
  commands:
    - ncem2png --help
```

**Comment by tschoonj** [comment]
@chrisburr thanks for the suggestions: they have been implemented accordingly. Could you restart the macOS build? It looks like something went wrong with the download from PyPI...

Thanks

**Comment by chrisburr** [comment]
[There is now a bot command you can use.](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci)

@conda-forge-admin, please restart ci

**Comment by tschoonj** [comment]
Yep looks like PyPI is having some trouble

**Comment by tschoonj** [comment]
@conda-forge-admin, please restart ci

**Comment by tschoonj** [comment]
Looks like PyPI is back in the game....

**Pull request merged**